### PR TITLE
fix gsg docs

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Thanks!

File: docs/source/getting-started/first-feature.rst [hunk 1]
```diff
@@ -78,11 +78,6 @@ Open a new terminal and run the following commands::
 
 The result is an up-to-date TypeScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
```
Comment (hurryabit) [inline docs/source/getting-started/first-feature.rst:79]: While we're at it, shouldn't it be like this?
```suggestion
The result is an up-to-date JavaScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
```

File: docs/source/getting-started/first-feature.rst [hunk 2]
```diff
@@ -78,11 +78,6 @@ Open a new terminal and run the following commands::
...
 
-To make sure that Yarn picks up the newly generated JavaScript code,
-we have to run the following command in the ``ui`` directory::
-
-  npm install --frozen-lockfile
-
```
Comment (hurryabit) [inline docs/source/getting-started/first-feature.rst:85]: Should we mention that `npm start` will pick up the changes automatically or would that only cause confusion for users who didn't even think about this problem in the first place?

**Review by hurryabit — APPROVED** [review]
Awesome. Thank you so much!

File: docs/source/getting-started/first-feature.rst (hunk 2 above)
Comment (garyverhaegen-da) [reply docs/source/getting-started/first-feature.rst:85]: I'd say deciding that goes a bit beyond the quickfix on a patched release, unless someone else has a strong opinion either way.

Comment (garyverhaegen-da) [reply docs/source/getting-started/first-feature.rst:85]: (And can express it right now.)

Comment (hurryabit) [reply docs/source/getting-started/first-feature.rst:85]: Let's skip it then. I guess it wouldn't be too helpful anyway.

File: docs/source/getting-started/first-feature.rst (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/source/getting-started/first-feature.rst:79]: Not in this fix.

**Review by cocreature — APPROVED** [review]
approving again since this has changed a few times in between

**Comment by garyverhaegen-da** [comment]
> approving again since this has changed a few times in between

Sorry about that. I'll try to remain disciplined next time.

**Comment by cocreature** [comment]
> Sorry about that. I'll try to remain disciplined next time.

Nothing to be sorry for, I just wanted to make it clear that the current state still has my approval to avoid any confusion :)

**Comment by hurryabit** [comment]
I just checked, we don't need to forward port any of these changes since the main branch does not have any of the issues address here anymore.

**Pull request merged**